examples/hello_world/android/settings.gradle

```diff
@@ -6,16 +6,29 @@
 // To update all the settings.gradle files in the Flutter repo,
 // See dev/tools/bin/generate_gradle_lockfiles.dart.
 
+pluginManagement {
+    def localPropertiesFile = new File(rootProject.projectDir, "local.properties")
+    def properties = new Properties()
+    assert localPropertiesFile.exists()
+    localPropertiesFile.withReader("UTF-8") { reader -> properties.load(reader) }
+    def flutterSdkPath = properties.getProperty("flutter.sdk")
+    assert flutterSdkPath != null, "flutter.sdk not set in local.properties"
+
+    includeBuild("$flutterSdkPath/packages/flutter_tools/gradle")
+
+    plugins {
+        id "dev.flutter.flutter-gradle-plugin" version "1.0.0" apply false
```

Comment:
Yup, done.

Should I update the template files as well? So that all new projects are generated with the declarative-style apply? Then this comment could be added there – the default apps people get after they `flutter create`  contain lots of code comments, so this wouldn't be too strange.

Reply:
Yes but if you want that can be a seperate PR.

Reply:
I migrated the templates in this PR (as I wished in the small todo list in [this comment](https://github.com/flutter/flutter/pull/123511#issuecomment-1494973039)).